Languages/IronPython/IronPythonCompiler/Program.cs

```diff
@@ -129,6 +129,34 @@ class Program {
             // variables for saving original working directory and return code of script
             var strVar = gen.DeclareLocal(u.Import(typeof(string)));
             var intVar = gen.DeclareLocal(u.Import(typeof(int)));
+            LocalBuilder dictVar = null;
+
+            if (config.PythonOptions.Count > 0) {
+                FieldInfo True = u.Import(typeof(ScriptingRuntimeHelpers)).GetField("True");
+                FieldInfo False = u.Import(typeof(ScriptingRuntimeHelpers)).GetField("False");
```

Comment:
Should we prefer `var` instead of the specifc name? And what about using `@true` or `_true`?